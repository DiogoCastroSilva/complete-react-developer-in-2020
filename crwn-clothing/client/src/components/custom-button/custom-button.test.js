import React from 'react';
import { shallow } from 'enzyme';

import CustomButton from './custom-button.component';


describe('testing CustomButon component', () => {
    it('should render the CustomButton', () => {
        expect(shallow(<CustomButton>Button</CustomButton>)).toMatchSnapshot();
    });
});