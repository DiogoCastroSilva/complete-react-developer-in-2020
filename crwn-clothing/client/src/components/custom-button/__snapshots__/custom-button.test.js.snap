// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing CustomButon component should render the CustomButton 1`] = `
<styled.button>
  Button
</styled.button>
`;
